src/core-services/product/mutations/updateProduct.js

```diff
@@ -92,6 +92,10 @@ const inputSchema = new SimpleSchema({
     type: String,
     optional: true
   },
+  "shouldAppearInSitemap": {
```

Comment:
Do we have any patterns that we follow for how we name boolean variables? 
The word `should` here feels a little ambiguous.

Reply:
To expand on this, we use `isDeleted` and `isVisible` above. Would this variable be better as `isInSitemap`?

Reply:
we do and this is one of them.

Reply:
This is an existing property of the product schema. It wasn't exposed in GraphQL queries/mutations. "should" is allowed for boolean values, along with "is". This convention seem to be inforced for React components. I don't always see it inforced for other types of JS.

Reply:
this seems ok to me